manifests/daemon.pp

```diff
@@ -129,9 +129,7 @@
       }
     }
     'none': {}
-    default: {
-      fail("The provided install method ${install_method} is invalid")
-    }
+    default: {}
```

Comment:
do we still need a default case, if we exhaust all enum options?

Reply:
in theory not. but puppet-lint demands one. If we remove it we need to add a comment to disable the puppet-lint check :(

Reply:
sounds like a bug in puppet lint